include/GafferSceneUI/UVView.h

```diff
@@ -127,6 +127,10 @@ class GAFFERSCENEUI_API UVView : public GafferUI::View
 
 		bool m_framed;
 
+		using DisplayTransformMap = std::unordered_map<std::string, IECoreGL::Shader::SetupPtr>;
```

Comment:
Do we have any concerns about the GPU memory usage of caching currently-unused transforms and their textures?

Reply:
I have switched this to caching the ImageProcessor, and regenerating the shader every time ( matching more closely to what was previously happening in TextureGadget ).  Thinking about this though, I should probably flag that currently this is only supporting GPU transforms.  If we decide to move the CPU transform to ImageView, there will be no way for UVView to use the CPU path, since it doesn't use ImageView.